etcdserver/server.go

```diff
@@ -81,6 +81,9 @@ const (
 	maxInFlightMsgSnap = 16
 
 	releaseDelayAfterSnapshot = 30 * time.Second
+
+	// Number of goroutines used in processing expired leases
```

Comment:
go comments follow [godoc](https://blog.golang.org/godoc-documenting-go-code), which start with the variable/function name and are full sentences:
`// maxPendingRevokes is the maximum number of outstanding expired lease revocations.`